### DISCUSSION TITLE
Point or cell highlighting after picking it

### DISCUSSION THREAD

**Discussion by karelvrabeckv** [Kitware/trame#467] [Trame/VTK] [answered]
Hello,
I have implemented a point/cell picker (using `vtkCellPicker`). When a user picks a specific point/cell, some text information about it appears (e.g., id, position, etc.). It is a little bit user-unfriendly now because, after a while, I cannot remember which point/cell I have picked. Therefore, I have an idea to highlight the point/cell after the user clicks on it, and I am a bit struggling with it.

Now:
![highlighting1](https://github.com/Kitware/trame/assets/62759688/0cbcd2a6-8d22-4cc6-8b48-838226de6d8d)

Intended:
![highlighting 2](https://github.com/Kitware/trame/assets/62759688/327ff7a2-93c9-4f4e-8154-9f958e492330)

**Is it possible to implement this highlighting using Trame/VTK?**
**Are there any ideas or approaches on how to implement this?**

Since I have access to a picked point/cell, I can change its properties, but I don't want to modify that data. Can it be implemented by (for example) creating copies of picked points/cells and coloring + rendering their edges only?

Thank you very much! :)

**Comment by jourdain** [comment, marked as answer]
Yes, you should just create a new polydata>mapper>actor using the cell information. Then you set the line width bigger like on your render.
HTH
